case-study/sqls/working-hour-distribution/sql

```diff
@@ -0,0 +1 @@
+SELECT COUNT(*) AS count, toDayOfWeek(created_at) AS dayOfWeek, toHour(created_at) AS hour FROM {{table}} GROUP BY hour, dayOfWeek
```

Comment:
I didn't see `repoIdList` here and you need to give working hour distribution for every repo I think.

Reply:
Oops! My fault, and I forgot to commit the newest changes.